docs/references/general/grid.md

```diff
@@ -0,0 +1,103 @@
+<!-- Copyright (C) 2020 SuperDARN Canada, University of Saskatchewan 
+author(s): Marina Schmidt 
+
+Disclaimer: License under GNU v3.0, the file is found in the root directory under LICENSE 
+
+-->
+# GRID files 
+
+GRID files are post-processed data product from FITACF files.
+
+## Naming Conventions
+
+### Single GRID files
+
+Currently the common naming convention for a GRID file is:
+
+> YYYYMMDD.HH.mm.ss.<3-letter abbreviation>.grid
+
+SuperDARN radars routinely change operating frequency. Some operating modes use multiple frequencies, either at the same time, or by alternating between frequencies. As a result, some files from some radars include all records for all frequencies in the same file and other radars separate out the individual frequencies into separate files, designated by a channel letter in the file name:
+
+> YYYYMMDD.HH.mm.ss.<3-letter abbreviation>.[a-d].grid
+
+See [fitacf](fitacf.md) documentation for more information on this naming convention. 
+
+If a GRID file is produced for 24-hours using the `-c` option in `make_grid` (see [`make_grid` tutorial](../../user_guide/make_grid.md)) then the common naming convention is:
+
+> YYYYYMMDD.<3-letter abbreviation>.grid
+
+### Combined GRID files 
+
+After GRID files are produced for seperate radars, they are sometimes further combined with other radars GRID files to meka a GRD file. Similar to GRID file just with multiple radar data included. 
```

Comment:
```suggestion
After GRID files are produced for separate radars, they are sometimes further combined with other radars GRID files to make a GRD file. This file has the same format as a GRID file, but includes data from multiple radars. 
```

I also recommend specifying that separate files are created for the northern and southern hemispheres, and by default all available radars in a given hemisphere are included.

Reply:
Also, I just noticed that GRD and GRID seem to be used interchangeably in this paragraph/document. Is there a particular reason for this?

Reply:
@mtwalach This is explained at the beginning of this document. GRID=gridded data file for a single radar, produced by `make_grid`. GRD=file containing gridded data from multiple radars, produced by `combine_grid`

Reply:
From where did the GRD vs GRID distinction originate?  Was it for the purposes of this document?

Reply:
It can be, please make suggestions and stuff. This is just a starting templated based on the old docs.

Reply:
Ah sorry, I didn't see that because github was only showing me changes that had been made, not the whole document (forever learning).

I am also curious about where this distinction comes from? Do we need one? Is it useful to have?

Reply:
I think @aburrell's suggestion here is good. I will commit that and then make a further suggestion